salt/top.sls

```diff
@@ -14,11 +14,17 @@ base:
     - flannel
     - docker
     - haproxy
+    - kube-common
```

Comment:
Generally, I would try to avoid applying "boring" roles like kube-common to a set of servers, and instead allow the other that need it (e.g.kube-apiserver) to include it.

This is really just a matter of taste though, I believe the top.sls (or playbooks in ansible..)  should show only the important bits - i.e. what services are getting deployed..